app/components/agenda-manager/index.js

```diff
@@ -13,35 +11,20 @@ import {task} from "ember-concurrency-decorators";
 
 /** @extends {Component<Args>} */
 export default class AgendaManagerIndexComponent extends Component {
-  /** @type AgendaData */
-  @tracked popup = false;
-  @tracked editModalVisible = false;
```

Comment:
Deriving the state from the `itemToEdit` prop seemed more logical since the edit modal will never be used if that prop is empty. Keeping a separate flag and keeping them in sync seemed unnecessary.

Reply:
makes sense